setup.py

```diff
@@ -591,5 +594,144 @@ def do_setup():
     )
 
 
+PROVIDERS_DEPENDENCIES = {
+    "amazon": [aws],
+    "apache.cassandra": [cassandra],
+    "apache.druid": [druid],
+    "apache.hdfs": [hdfs],
+    "apache.hive": [hive],
+    "apache.pig": [],
+    "apache.pinot": [pinot],
+    "apache.spark": [],
+    "apache.sqoop": [],
+    "celery": [celery],
+    "cloudant": [cloudant],
+    "cncf.kubernetes": [kubernetes],
+    "databricks": [databricks],
+    "datadog": [datadog],
+    "dingding": [],
+    "discord": [],
+    "docker": [docker],
+    "email": [],
+    "ftp": [],
+    "google.cloud": [gcp],
+    "google.marketing_platform": [gcp],
+    "google.suite": [gcp],
+    "grpc": [grpc],
+    "http": [],
+    "imap": [],
+    "jdbc": [jdbc],
+    "jenkins": [jenkins],
+    "jira": [jira],
+    "microsoft.azure": [azure],
+    "microsoft.mssql": [mssql],
+    "microsoft.winrm": [winrm],
+    "mongo": [mongo],
+    "mysql": [mysql],
+    "odbc": [odbc],
+    "openfass": [],
+    "opsgenie": [],
+    "oracle": [oracle],
+    "pagerduty": [pagerduty],
+    "papermill": [papermill],
+    "postgres": [postgres],
+    "presto": [presto],
+    "qubole": [qds],
+    "redis": [redis],
+    "salesforce": [salesforce],
+    "samba": [samba],
+    "segment": [segment],
+    "sftp": [],
```

Comment:
```suggestion
    "sftp": [ssh],
```

SFTP needs the same deps as ssh

Reply:
Right!